app/feature/feature-edit-coinsured/src/main/kotlin/com/hedvig/android/feature/editcoinsured/ui/EditCoInsuredPresenter.kt

```diff
@@ -184,6 +192,31 @@ internal class EditCoInsuredPresenter(
       }
     }
 
+    if (commit) {
+      LaunchedEffect(commit) {
+        intentId?.let {
```

Comment:
`intentId` is used inside the LaunchedEffect here, but it's not part of the keys of `LaunchedEffect` this will mean that if `intentId` changes, this effect will not be run again, is this something we want here?

Reply:
Yeah this should be run based on if you want to commit the intentid, so we dont want the key to be the intentid itself